src/main/scala/ru/d4team/chat/services/PersonService.scala

```diff
@@ -0,0 +1,26 @@
+package ru.d4team.chat.services
+
+import ru.d4team.chat.dao.PersonDAO
+import ru.d4team.chat.models.person.Person
+import zio._
+
+import java.util.UUID
+
+object PersonService {
+  val live: URLayer[PersonDAO, PersonService] = ZLayer.fromFunction(new PersonServiceImpl(_))
+}
+
+trait PersonService {
+  def addPerson(person: Person): Task[Person]
+  def findPerson(id: UUID): Task[Option[Person]]
+  def getAll: Task[List[Person]]
+}
+
+final case class PersonServiceImpl(dao: PersonDAO) extends PersonService {
+  override def addPerson(person: Person): Task[Person] = dao.addPerson(person)
+
+  override def findPerson(id: UUID): Task[Option[Person]] =
+    dao.findPerson(id)
+
+  override def getAll: Task[List[Person]] = dao.getPersons
+}
```

Comment:
Also, to be consistent with other services, let's use this order:
1. contract
2. object
3. implementation